modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/index.d.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ReactElement} from 'react';
+
+import type {Atom} from './State';
```

Comment:
I don't think the `./State` file exists does it?

Reply:
You may also need to add `@liferay/frontend-js-state-web` to this module's package.json